src/indigo/components/SearchBar.jsx

```diff
@@ -1,52 +1,35 @@
 import React from 'react';
```

Comment:
@ujovlado muzes prosim mrknout na moji prvni stateless komponentu jestli jsem ji prepsal dobre?

Reply:
jo, kuknem na to potom ... btw, stateless by bola keby nemala state ;)

Reply:
ajo. funkcionalni komponenta?

Reply:
nic funkcionalniho tam nevidim :) rekl bych proste komponenta :)

Reply:
vracias tam classu (`export default class .... `), nie funkciu ... je to tak ako pise Martin
https://reactjs.org/docs/components-and-props.html#functional-and-class-components

Reply:
Fajn, tak uz vim jak se to NEjmenuje .]